node_package/tests/ReactOnRails.test.js

```
@@ -6,7 +6,7 @@
 import { createStore } from 'redux';
 import * as React from 'react';
 import * as createReactClass from 'create-react-class';
-import ReactOnRails from '../src/ReactOnRails';
+import ReactOnRails from '../src/ReactOnRails/client';
```

ESLint

[error] Missing file extension for "../src/ReactOnRails/client"

(import/extensions)